api_server/src/main/java/io/zetch/app/domain/review/ReviewPostDto.java

```diff
@@ -0,0 +1,18 @@
+package io.zetch.app.domain.review;
+
+import java.io.Serializable;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.ToString;
+import lombok.Value;
+
+@Value
+@Builder
+@AllArgsConstructor
+public class ReviewPostDto implements Serializable {
```

Comment:
Is this different than ReviewDto

Reply:
I thought about passing only `location_id`, `user_id` in POST `/reviews`, but fetching concrete `location`, `user` for the GET `/reviews`.

I now see that we can make it more concise if we define the base dto, `ReviewDto` that includes `comment` and `rating`, and then make `ReviewPostDto` and `ReviewGetDto` inherited by that.

What do you think about this idea?